app/src/main/java/ch/hikemate/app/ui/guide/GuideScreen.kt

```diff
@@ -0,0 +1,286 @@
+              .testTag(GuideScreen.GUIDE_HEADER)) {
+        // HikeMate's logo
+        Image(
+            painter = painterResource(id = R.drawable.app_icon),
+            contentDescription = stringResource(R.string.guide_app_logo),
+            modifier = Modifier.size(GuideScreen.ICON_SIZE_DP.dp))
+        Spacer(modifier = Modifier.width(GuideScreen.HEADER_SPACING_DP.dp))
+        Text(
+            text = stringResource(R.string.guide_title),
+            style = MaterialTheme.typography.headlineMedium)
+      }
+}
+
+@Composable
+private fun HikingGuideSection() {
+  Spacer(modifier = Modifier.height(GuideScreen.SECTION_SPACING_DP.dp))
+  Text(
+      text = stringResource(R.string.guide_hiking_section_title),
+      style = MaterialTheme.typography.headlineMedium)
+  Spacer(modifier = Modifier.height(GuideScreen.HEADER_SPACING_DP.dp))
+}
+
+/**
+ * The composable function for a single topic card
+ *
+ * @param topic the topic which will be described
+ * @param navigationActions
+ */
+@Composable
+private fun ExpandableTopicCard(
+    topic: GuideTopic,
+    navigationActions: NavigationActions,
+) {
+  var isExpanded by remember { mutableStateOf(false) }
+
+  Card(
+      modifier =
+          Modifier.fillMaxWidth()
+              .padding(vertical = GuideScreen.CARD_SPACING_DP.dp)
+              .animateContentSize()
+              .testTag("${GuideScreen.TOPIC_CARD}_${topic.titleResId}"),
+      shape = RoundedCornerShape(GuideScreen.CORNER_RADIUS_DP.dp)) {
+        TopicCardContent(
+            topic = topic,
+            isExpanded = isExpanded,
+            onExpandToggle = { isExpanded = !isExpanded },
+            navigationActions = navigationActions)
+      }
+}
+
+@Composable
+private fun TopicCardContent(
+    topic: GuideTopic,
+    isExpanded: Boolean,
+    onExpandToggle: () -> Unit,
+    navigationActions: NavigationActions
+) {
+  Column {
+    // Move clickable to just the header
+    Row(
+        modifier =
+            Modifier.fillMaxWidth()
+                .clickable(onClick = onExpandToggle)
+                .testTag("${GuideScreen.TOPIC_HEADER}_${topic.titleResId}")
+                .padding(GuideScreen.CONTENT_PADDING_DP.dp),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically) {
+          Text(
+              text = stringResource(topic.titleResId), style = MaterialTheme.typography.titleMedium)
+          Icon(
+              imageVector =
+                  if (isExpanded) Icons.Default.KeyboardArrowUp
+                  else Icons.Default.KeyboardArrowDown,
+              contentDescription =
+                  stringResource(
+                      if (isExpanded) R.string.collapse_topic else R.string.expand_topic))
+        }
+    ExpandableContent(topic = topic, isExpanded = isExpanded, navigationActions = navigationActions)
+  }
+}
+
+@Composable
+private fun ExpandableContent(
```

Comment:
Warning: Please add a docstring to this composable. This makes the code easier to maintain.